WISE-Community/WISE-Client
# Remove upgrade from summary display

- Test with https://github.com/WISE-Community/WISE-API/pull/142

## Changes

- Created teacher summary display
- Created student summary display
- Removed upgrade dependency
- Preview component now accepts a period id parameter

## Test

- Make sure students can retrieve classmate summary student work and scores for period and class
- Make sure teachers can retrieve classmate summary student work and scores for period and class
  - You can view summary displays in the Classroom Monitor Node Info if there is a Multiple Choice component or a component that has been scored
  - You can also view summary displays in the Classroom Monitor Node Info if there is a Summary component authored into the step

Closes #639
